### ISSUE TITLE
avatar command no longer works 

### ISSUE THREAD

**Issue by ajloveslily14** [jagrosh/Selfbot#37]
When you use (prefix).avatar @usermention#discrim the following error gets spit into console:

```
[18:03:27] [Fatal] [JDA]: One of the EventListeners had an uncaught exception
[18:03:27] [Fatal] [JDA]: Encountered an exception:
[18:03:27] [Fatal] [JDA]: java.lang.NumberFormatException: The specified ID is not a valid snowflake (<@90357086428368896>). Expecting a valid long value!
        at net.dv8tion.jda.core.utils.MiscUtil.parseSnowflake(MiscUtil.java:137)
        at net.dv8tion.jda.core.entities.impl.JDAImpl.getGuildById(JDAImpl.java:407)
        at jselfbot.commands.AvatarCmd.execute(AvatarCmd.java:51)
        at jselfbot.Command.run(Command.java:42)
        at jselfbot.Bot.onMessageReceived(Bot.java:114)
        at net.dv8tion.jda.core.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:381)
        at net.dv8tion.jda.core.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:84)
        at net.dv8tion.jda.core.handle.MessageCreateHandler.handleDefaultMessage(MessageCreateHandler.java:129)
        at net.dv8tion.jda.core.handle.MessageCreateHandler.handleInternally(MessageCreateHandler.java:50)
        at net.dv8tion.jda.core.handle.SocketHandler.handle(SocketHandler.java:38)
        at net.dv8tion.jda.core.requests.WebSocketClient.handleEvent(WebSocketClient.java:770)
        at net.dv8tion.jda.core.requests.WebSocketClient.onTextMessage(WebSocketClient.java:500)
        at com.neovisionaries.ws.client.ListenerManager.callOnTextMessage(ListenerManager.java:352)
        at com.neovisionaries.ws.client.ReadingThread.callOnTextMessage(ReadingThread.java:260)
        at com.neovisionaries.ws.client.ReadingThread.callOnTextMessage(ReadingThread.java:238)
        at com.neovisionaries.ws.client.ReadingThread.handleTextFrame(ReadingThread.java:963)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:746)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
```

and the command fails

**Comment by jagrosh** [comment]
Can you confirm that this has been fixed as of [Version 0.9](https://github.com/jagrosh/Selfbot/releases/tag/0.9)?

**Comment by ajloveslily14** [comment]
That seems to have fixed it, thanks!
